src/arithmetic.jl

```diff
@@ -201,10 +203,12 @@ According to the IEEE-1788 standard:
 When `∘` is commutative, these agree and we write `∘_rev(b, c, x)`.
 """
 
-function mul_rev_IEEE1788(b, c, x)   # c = b*x
-    return x ∩ (c / b)
+function mul_rev_IEEE1788(b, c, x)   
+    return x ∩ (extended_div(c,b)[1] ∪ extended_div(c,b)[2])
```

Comment:
You are calculating `extended_div(c, b)` twice. 

Please always add a space after a comma.